### PR TITLE
Multi cloud support config:ansible-tower-implementation

### PR DESCRIPTION
**Pull request by prakhar1985**
Summary 

Multi cloud support enhanced 

Issue:

1: Unused files deleted from config.
2: Using new variables file structor for multi cloud provider support.

### CODE REVIEW

File: ansible/configs/ansible-tower-implementation/default_vars.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# rhsm_pool_ids:
+#   - 8a85f99b6b498682016b521dfe463949
+# rhel_subscription_user: 
+# rhel_subscription_pass: 
+### If using repo_method: satellite, you must set these values as well.
+# satellite_url: satellite.example.com
+# satellite_org: Sat_org_name
+# satellite_activationkey: "rhel7basic"
+######
+repo_version: "3.6.0-1"
```
Comment (stencell) [inline ansible/configs/ansible-tower-implementation/default_vars.yml]: You have `repo_version` var defined twice. See line 18 also.

File: ansible/configs/ansible-tower-implementation/default_vars_ec2.yml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+# This is the Route53 HostedZoneId where you will create your Public DNS entries
+# This only needs to be defined if your CF template uses route53
+HostedZoneId: Z3IHLWJZOU9SRT
+aws_region: ap-southeast-2
+
+ansible_user: ec2-user
+remote_user: ec2-user
+
+## Environment Sizing
+bastion_instance_type: "t2.medium"
```
Comment (stencell) [inline ansible/configs/ansible-tower-implementation/default_vars_ec2.yml]: Not required, but consider moving to t3 to be more future proof.

File: ansible/configs/ansible-tower-implementation/default_vars.yml (hunk 1 above)
Comment (prakhar1985) [reply ansible/configs/ansible-tower-implementation/default_vars.yml]: Fixed

File: ansible/configs/ansible-tower-implementation/default_vars_ec2.yml (hunk 2 above)
Comment (prakhar1985) [reply ansible/configs/ansible-tower-implementation/default_vars_ec2.yml]: changed to t3

**Comment by prakhar1985** [comment]
@stencell Please review and merge.

**Comment by prakhar1985** [comment]
@stencell can you please merge?

**Pull request merged**